server/server.js

```diff
@@ -1,7 +1,28 @@
-// Copyright (C) 2012-present, The Authors. This program is free software: you can redistribute it and/or  modify it under the terms of the GNU Affero General Public License, version 3, as published by the Free Software Foundation. This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more details. You should have received a copy of the GNU Affero General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+// Copyright (C) 2012-present, The Authors. This program is free software: you
+// can redistribute it and/or  modify it under the terms of the GNU Affero
+// General Public License, version 3, as published by the Free Software
+// Foundation. This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+// General Public License for more details. You should have received a copy of
+// the GNU Affero General Public License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
 
 "use strict";
 
+// for some reason, this cannot be done in the dockerfile
+const childProcess = require('child_process');
+childProcess.execSync('cp /config/* ./config');
```

Comment:
Is there another way to get this working than copying files in a subprocess and launch?